ddtrace/tracer/log_test.go

```diff
@@ -52,10 +53,19 @@ func TestStartupLog(t *testing.T) {
 		defer globalconfig.SetServiceName("")
 		defer stop()
 
+		//Override longrunner var to get around not having an `/info` endpoint
```

Comment:
```suggestion
		// Override longrunner var to get around not having an `/info` endpoint
```
Please do this everywhere.